IDT.XGen/Tests/UI/StepDefinitions/Distributor/RegisterRetailersSteps.cs

```
@@ -73,5 +73,38 @@ public void ThenTheRegisterRetailerShouldDisplayTheMessage(string message)
             string expectedResult = this.registerRetailerPage.GetMessage();
             Assert.AreEqual(expectedResult, message);
         }
+
+        [StepDefinition(@"I switch to ""(.*)"" Wallet")]
+        public void WhenISwitchToWallet(string switchTo)
+        {
+            this.homePage = new HomePage();
+            switch (switchTo)
+            {
+                case "Default":
+                    this.homePage.ClickChkDefaultWallet();
+                    break;
+                case "Regulated":
+                    this.homePage.ClickChkRegulatedWallet();
+                    break;
+                default:
+                    break;
```

Here we should raise an exception